code/modules/mob/living/simple_animal/hostile/wastebots.dm

```diff
@@ -70,3 +74,84 @@
 	projectiletype = /obj/item/projectile/beam/laser/pistol
 	faction = list("wastebot")
 	check_friendly_fire = 1
+
+/mob/living/simple_animal/hostile/handy/sentrybot
+	name = "sentry bot"
+	desc = "A pre-war military robot armed with a deadly gatling laser and covered in thick armor plating."
+	icon_state = "sentrybot"
+	icon_living = "sentrybot"
+	icon_dead = "sentrybot"
+	health = 360
+	maxHealth = 360
+	healable = 0
+	speed = 1
+	melee_damage_lower = 85
+	melee_damage_upper = 85
+	rapid = 2 //6 projectiles. oof.
+	ranged_cooldown_time = 15 //brrrrrrrrrrrrt
+	ranged = 1
+	retreat_distance = 0
+	minimum_distance = 0
+	attacktext = "pulverizes"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	projectilesound = 'sound/weapons/laser.ogg'
+	projectiletype = /obj/item/projectile/beam/laser/pistol/weak
+	faction = list("wastebot")
```

Comment:
```suggestion
```
Default value.

Reply:
This should be already inherited for the parent. Are you keeping it for code clarity?